doc/Programming-guide/Key-platform-features/Endpoint-groups/index.md

```diff
@@ -274,16 +286,17 @@ To add a new endpoint group, do the following:
 
 1. Open the **Endpoint groups** window by clicking **Endpoint groups** under the application menu on the navigation panel and then click **Add endpoint group**.
 
-<img src="admin-ui/endpoint-groups.png">
+![endpoint-groups](admin-ui/endpoint-groups.png "endpoint-groups")
+
 
 2. In the **Add endpoint group** window, fill in the required fields and then click **Add**.
 
-<img src="admin-ui/create-endpoint-group.png">
+![create-endpoint-group](admin-ui/create-endpoint-group.png "create-endpoint-group")
```

Comment:
Don't add stupid alt text and title. Please, read https://en.wikipedia.org/wiki/Alt_attribute before continuing.